tests/e2e/integration/Explorer/main.js

```diff
@@ -0,0 +1,278 @@
+            .contains('bits')
+            .click();
+          cy.get(homepage.units.units).should('contain', 'bits');
+        });
+
+        it('should change currency unit to sat', () => {
+          cy.get(homepage.header.navBar)
+            .click()
+            .get(homepage.header.navlist)
+            .contains('sat')
+            .click();
+          cy.get(homepage.units.units).should('contain', 'sat');
+        });
+
+        it('should change currency unit to usd', () => {
+          cy.get(homepage.header.navBar)
+            .click()
+            .get(homepage.header.navlist)
+            .contains('usd')
+            .click();
+          cy.get(homepage.units.units).should('contain', 'usd');
+        });
+
+        it('should change currency unit to euro', () => {
+          cy.get(homepage.header.navBar)
+            .click()
+            .get(homepage.header.navlist)
+            .contains('euro')
+            .click();
+          cy.get(homepage.units.units).should('contain', 'euro');
+        });
+      });
+
+      describe('Changing background color', () => {
+        it('should start with light background', () => {
+          cy.get('body').should(
+            'have.css',
+            'background-color',
+            'rgb(52, 58, 64)'
+          );
+        });
+
+        it('should change background color to dark', () => {
+          cy.get(homepage.header.navBar)
+            .click()
+            .get(homepage.navbarTheme.dark)
+            .click()
+            .should('have.css', 'background-color', 'rgba(0, 0, 0, 0)');
+        });
+      });
+
+      describe('Searching data', () => {
+        it('should find latest block in searchbar', () => {
+          cy.get(homepage.latestBlocks.height)
+            .invoke('text')
+            .then(($height) => {
+              const myVar = $height.replace(',', '');
+              cy.get(homepage.header.searchBar).type(`${myVar}{enter}`);
+              cy.get(blockheight.details.jsonTab)
+                .click()
+                .get(blockheight.jsonTab.blockHeight)
+                .invoke('text')
+                .then(($blockHeight) => {
+                  expect(myVar).to.eq($blockHeight);
+                });
+            });
+        });
+
+        it('should find block hash in searchbar', () => {
+          cy.get(homepage.latestBlocks.height).click();
+          cy.get(blockheight.details.hash)
+            .invoke('text')
+            .then(($hash) => {
+              cy.get(homepage.header.searchBar).type(`${$hash}{enter}`);
+              cy.get(blockheight.details.jsonTab).click();
+              cy.get(blockheight.jsonTab.hash)
+                .invoke('text')
+                .then(($hash1) => {
+                  const myVar = $hash1.replace(/['"]+/g, '');
+                  expect(myVar).to.eq($hash);
+                });
+            });
+        });
+
+        // it('should find transaction id in searchbar', () => {
```

Comment:
If it's only temporary, please leave above this comment block a line with TODO comment. Example:
```
  // TODO: Uncomment it once the feature is working
  // it('should find transaction id in searchbar', () => {
```

Otherwise, remove it if unnecessary.